cmd/lima-guestagent/install_systemd_linux.go

```diff
@@ -97,6 +118,15 @@ func generateSystemdUnit(vsockPort int, virtioPort string, debug bool) ([]byte,
 	if debug {
 		args = append(args, "--debug")
 	}
+	if len(dockerSockets) > 0 {
+		args = append(args, fmt.Sprintf("--docker-sockets %s", strings.Join(dockerSockets, ",")))
```

Comment:
I guess it works because it is all stringified again, but it feels odd to squeeze 2 arguments into one slice element.

```suggestion
		args = append(args, "--docker-sockets", strings.Join(dockerSockets, ","))
```